tests/unit_tests/test_image_series.py

```diff
@@ -131,11 +134,73 @@ def test_check_large_image_series_stored_internally():
 
     expected_message = InspectorMessage(
         importance=Importance.BEST_PRACTICE_VIOLATION,
-        message=f"ImageSeries {image_series.name} is too large. Use external mode for storage",
+        message="ImageSeries is very large. Consider using external mode for better storage.",
         check_function_name="check_image_series_data_size",
         object_type="ImageSeries",
         object_name="ImageSeriesLarge",
         location="/",
```

Comment:
The name of the `ImageSeries` object is returned in the `InspectorMessage`, so that part of the string `message` is duplicated.